packages/feedback/src/modal/integration.tsx

```diff
@@ -69,8 +69,8 @@ export const feedbackModalIntegration = ((): FeedbackModalIntegration => {
             screenshotInput={screenshotInput}
             showName={options.showName || options.isNameRequired}
             showEmail={options.showEmail || options.isEmailRequired}
-            defaultName={(userKey && user?.[userKey.name]) || ''}
-            defaultEmail={(userKey && user?.[userKey.email]) || ''}
+            defaultName={String((userKey && user?.[userKey.name]) || '')}
```

Comment:
I guess this comes from the type change in `User`. Wondering if this would also apply to anyone outside of our SDK code. See comment above

Reply:
Yes, it does. I'll give it the same treatment.